ethcore/src/block.rs

```diff
@@ -498,7 +498,7 @@ pub fn enact(
 	tracing: bool,
 	db: Box<JournalDB>,
 	parent: &Header,
-	last_hashes: LastHashes,
+	last_hashes: Arc<LastHashes>,
```

Comment:
seems like all of these `Arc<LastHashes>` could just be an `&LastHashes` and we would save a few atomic ops in a bottleneck.